sdk/core/Azure.Core.TestFramework/src/TestRecording.cs

```diff
@@ -113,7 +113,8 @@ internal async Task InitializeProxySettingsAsync()
                     var excludedHeaders = new List<string>(_matcher.LegacyExcludedHeaders)
                     {
                         "Content-Type",
```

Comment:
Why did we ignore content type?

Reply:
Ah we should be able to remove this now that core has shipped. I'll do it in a follow-up PR.